docs/plugins/acousticbrainz.rst

```diff
@@ -14,9 +14,34 @@ Enable the ``acousticbrainz`` plugin in your configuration (see :ref:`using-plug
 For all tracks with a MusicBrainz recording ID, the plugin currently sets
 these fields:
 
-* ``danceable``: Predicts how easy the track is to dance to.
-* ``mood_happy``: Predicts the probability this track will evoke happiness.
-* ``mood_party``: Predicts the probability this track should be played at a
-  party.
+* ``danceable``
+* ``gender``
+* ``genre_rosamerica``
+* ``mood_acoustic``
+* ``mood_aggressive``
+* ``mood_electronic``
+* ``mood_happy``
+* ``mood_party``
+* ``mood_relaxed``
+* ``mood_sad``
+* ``rhythm``
+* ``tonal``
+* ``voice_instrumental``
```

Comment:
Perhaps we should link to the AcousticBrainz docs to explain what these fields mean?